innovation.scss

```diff
@@ -1144,36 +1110,14 @@
         &.icon_6 {
             background-position: -95px -82px;
         }
-        &.age_1 {
-            background-position: -0px -101px;
-        }
-        &.age_2 {
-            background-position: -19px -101px;
-        }
-        &.age_3 {
-            background-position: -38px -101px;
-        }
-        &.age_4 {
-            background-position: -57px -101px;
-        }
-        &.age_5 {
-            background-position: -76px -101px;
-        }
-        &.age_6 {
-            background-position: -95px -101px;
-        }
-        &.age_7 {
-            background-position: -114px -101px;
-        }
-        &.age_8 {
-            background-position: -133px -101px;
-        }
-        &.age_9 {
-            background-position: -152px -101px;
-        }
-        &.age_10 {
-            background-position: -171px -101px;
-        }
+        &.age {
+			@include square-age;
+			font-size: inherit;
+			line-height: 19px;
+			&.age_10 {
+				@include square-age-10;
+			}
+		}
     }
```

Comment:
I need to add an in_tooltip age section as there is an 8 box in a special achievement tool tip